### PR TITLE
chore: migrate to pnpm

### PR DESCRIPTION
**Pull request by danielleroux**
### GitHub
- Refactor CONTRIBUTE.md
- Refactor README.md to look more modern
- Make Pull Request template more simple
- Remove git message lint validation
- Migrate from yarn to pnpm
- Remove gh-pages deployment in preparation of netlify deployment
- Add `changeset`

###  Docs
- Update to Docusaurus v3
- Migrate Docusaurus from javascript version to typescript version
- Change menu structure based on latest feedback

### `@siemens/ix-angular`
- Add `forms` dependency

## 🧪 Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

### CODE REVIEW

File: packages/documentation/docs/guidelines/theme.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ If this step is done you can set the theme name class on the body tag:
 
 The Siemens AG Corporate Brand Theme is exclusively available to official Siemens AG products where it should always be used as the default theme to reflect the latest Siemens AG Corporate Brand guidelines.
 
-Siemens AG employees can access the Corporate Brand Theme at [**https://code.siemens.com/siemens-ix/ix-brand-theme**](https://code.siemens.com/siemens-ix/ix-brand-theme).
+Siemens AG employees can access the Corporate Brand Theme [here](https://code.siemens.com/siemens-ix/ix-brand-theme).
```
Comment (nuke-ellington) [inline packages/documentation/docs/guidelines/theme.md:51]: This was changed to the full URL in a recent PR. Why revert? I like here better but it was not very clear for users :)

File: packages/documentation/package.json [hunk 2]
```diff
@@ -58,12 +63,12 @@
       "not op_mini all"
     ],
     "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
+      "last 3 chrome version",
+      "last 3 firefox version",
+      "last 5 safari version"
```
Comment (nuke-ellington) [inline packages/documentation/package.json:68]:
```suggestion
      "last 3 chrome versions",
      "last 3 firefox versions",
      "last 5 safari versions"
```

**Review by nuke-ellington — CHANGES_REQUESTED** [review]
Various comments

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -93,3 +299,13 @@ Must be one of the following:
 - **perf**: A code change that improves performance
 - **refactor**: A code change that neither fixes a bug nor adds a feature
 - **test**: Adding missing tests or correcting existing tests
+
+### Submit Pull Request
+
+1. [Create a new pull request](https://github.com/siemens/ix/compare) with the `main` branch as the `base`. You may need to click on `compare across forks` to find your changes.
+2. See the [Creating a pull request from a fork](https://help.github.com/articles/creating-a-pull-request-from-a-fork/) GitHub help article for more information.
+3. Please fill out the provided Pull Request template to the best of your ability and include any issues that are related.
+4. Create a [changesets](https://github.com/changesets/changesets/blob/main/docs/adding-a-changeset.md) to help us prepare a release. If you are not sure what to do just skip this step, a maintainer will comment on the pull request.
```
[the remark this answers is not in the document]
Comment (danielleroux) [reply CONTRIBUTING.md:308]: `changesets` is the name of the tool

File: packages/documentation/docs/guidelines/theme.md (hunk 1 above)
Comment (danielleroux) [reply packages/documentation/docs/guidelines/theme.md:51]: image

File: CONTRIBUTING.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (nuke-ellington) [reply CONTRIBUTING.md:311]: Obviously, but the link literally is called adding a changeset :)

**Review by nuke-ellington — APPROVED** [review]

**Review by nuke-ellington — APPROVED** [review]

File: .github/workflows/build.yml [hunk 4]
```diff
@@ -1,108 +1,104 @@
...
-      image: node:18.12.1
-      env:
-        DEBIAN_FRONTEND: noninteractive
+      image: mcr.microsoft.com/playwright:v1.39.0-jammy
     steps:
       - uses: actions/checkout@v3
-      - uses: ./.github/workflows/actions/lint
+      - uses: ./.github/workflows/actions/turbo
+      - name: Test
+        run: pnpm run test --cache-dir=.turbo --filter @siemens/ix --filter @siemens/ix-angular
```
Comment (nuke-ellington) [inline .github/workflows/build.yml:104]: We should add react if #1017 gets merged beforehand.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -4,66 +4,238 @@ SPDX-FileCopyrightText: 2023 Siemens AG
...
+Once it's filed:
 
-## Design specs/styleguide
+#### Issue workflow
 
-If you need access to iX design documentation please contact a iX maintainer.
+- The project team will label the issue accordingly. `Workflow: Issue created`
+- A team member will try to reproduce the issue with your provided steps. If there are no reproduction steps or no obvious way to reproduce the issue, the team will ask you for those steps and mark the issue as `needs-repro`. Bugs with the `needs-repro` label will not be addressed until the steps to reproduce it have been provided. It is necessary to provide accessible repository.
+- If the team is able to reproduce the issue, it will be marked `type: bug` and `Workflow: Approved`, as well as possibly other labels (such as `type: enhancement`)
+- If someone of the team started to work on the issue the workflow label will be changed to `Workflow: In progress`
```
Comment (nuke-ellington) [inline CONTRIBUTING.md:66]:
```suggestion
- If work on the issue has begun the workflow label will be changed to `Workflow: In progress`
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -4,66 +4,238 @@ SPDX-FileCopyrightText: 2023 Siemens AG
...
-- [Angular coding style guide](https://angular.io/guide/styleguide)
-- [Google's TypeScript style guide](https://google.github.io/styleguide/tsguide.html)
+Once it's filed:
 
-## Design specs/styleguide
+#### Issue workflow
 
-If you need access to iX design documentation please contact a iX maintainer.
+- The project team will label the issue accordingly. `Workflow: Issue created`
+- A team member will try to reproduce the issue with your provided steps. If there are no reproduction steps or no obvious way to reproduce the issue, the team will ask you for those steps and mark the issue as `needs-repro`. Bugs with the `needs-repro` label will not be addressed until the steps to reproduce it have been provided. It is necessary to provide accessible repository.
```
Comment (nuke-ellington) [inline CONTRIBUTING.md:64]:
```suggestion
- A team member will try to reproduce the issue with your provided steps. If there are no reproduction steps or no obvious way to reproduce the issue, the team will ask you for those steps and mark the issue as `needs-repro`. Bugs with the `needs-repro` label will not be addressed until the steps to reproduce it have been provided. Ideally you can provide a link to a running example (e.g. StackBlitz).
```

**Review by nuke-ellington — CHANGES_REQUESTED** [review]

**Review by nuke-ellington — APPROVED** [review]

File: .github/workflows/actions/push-remote/action.yml [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+    default: 'core'
+
+  branch-name: # id of input
+    default: 'ix'
+
+  commit_id:
+    description: 'commit id'
+
+  token:
+    description: 'commit id'
```
Comment (nuke-ellington) [inline .github/workflows/actions/push-remote/action.yml:14]: Is the commit ID also the token?

Comment (danielleroux) [reply .github/workflows/actions/push-remote/action.yml]: Fixed

**Review by nuke-ellington — APPROVED** [review]

**Pull request merged**